src/cozmo/pets.py

```diff
@@ -50,7 +50,7 @@
 from ._clad import _clad_to_game_anki
 
 
-#: Length of time to go without receiving an observed event before
+#: Number of seconds to go without receiving an observed event before
```

Comment:
Ideally the wording here would be similar to that in objects.py for OBJECT_VISIBILITY_TIMEOUT - adding that this is in seconds is good (that is already done in the other places), but I'm not sure there's big win of changing from:
"Length of time in seconds"
to
"Number of seconds"